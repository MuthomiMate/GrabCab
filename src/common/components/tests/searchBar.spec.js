import React from "react";
import { shallow } from "enzyme";
import SearchBar from "../searchBar";

describe("<SearchBar />", () => {

    it("renders without crashing", () => {
        const wrapper = shallow(<SearchBar/>);
        expect(wrapper).toMatchSnapshot();
    })
})