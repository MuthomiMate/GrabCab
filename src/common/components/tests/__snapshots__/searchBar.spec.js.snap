// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBar /> renders without crashing 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#0A0F10",
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "padding": 10,
        "paddingBottom": 0,
        "paddingTop": 0,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "justifyContent": "space-between",
        }
      }
    >
      <GooglePlaces
        id="from"
        placeholder="Where From ?"
      />
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "padding": 10,
        "paddingTop": 0,
      }
    }
  >
    <GooglePlaces
      id="to"
      placeholder="Drop Where ?"
    />
  </View>
</View>
`;
